src/test/java/tests/SmokeTest.java

```diff
@@ -0,0 +1,17 @@
+package tests;
+
+import baseEntities.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.MainPage;
+
+public class SmokeTest extends BaseTest {
+
+    @Test
+    public void entranceFormTest(){
+        MainPage newMainPage = new MainPage();
+        newMainPage.openPage();
+        newMainPage.enterAccount();
+        Assert.assertEquals(new MainPage().getLoginViaNickNameMessage(), "или через ник, e-mail");
```

Comment:
why do you use new MainPage() if you have already declared a variable on the 12 line?

Reply:
fixed